src/med_bench/utils/nuisances.py

```diff
@@ -0,0 +1,511 @@
+from itertools import combinations
```

Comment:
Please add a  module-level docstring.

Reply:
This has been addressed in commit 14374c26028c5a15b5ce4dd3e8cd907a83d83a37